CMakeLists.txt

```diff
@@ -40,10 +40,24 @@ else()
     set(HALST_EXCLUDE_FROM_ALL "EXCLUDE_FROM_ALL")
 endif()
 
+if (EMIL_HOST_BUILD)
+    include(FetchContent)
+
+    FetchContent_Declare(
+        cucumber-cpp-runner
+        GIT_REPOSITORY https://github.com/philips-software/amp-cucumber-cpp-runner.git
+        GIT_TAG        7a25c498de47eaa53692b583c75ea6a0c4ac807d # unreleased
```

Comment:
points to a commit in a draft pull request here: https://github.com/philips-software/amp-cucumber-cpp-runner/pull/69